### PR TITLE
Problem: Some keys used in mini-provisioner are outdated.

### PR DESCRIPTION
**Pull request by SwapnilGaonkar7**
Solution: Modified code to use new updated keys
Also, downloaded sample confstore file from https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/85000402/Mini-Provisioner+ConfStore+Keys and uploaded in our repo
Note: Modified file to include 'durability' and changes value for data and parity

### CODE REVIEW

**Review by knekrasov — APPROVED** [review]

File: provisioning/miniprov/hare_mp/cdf.py [hunk 1]
```diff
@@ -56,51 +56,62 @@ def _create_pool_descriptions(self) -> List[PoolDesc]:
...
+            )
+            spare_units_count = int(
+                conf.get(
+                    f'cluster>{cluster_id}>storage_set[{x}]>durability>spare'))
 
             min_pool_width = data_units_count + parity_units_count \
                 + spare_units_count
-            if (data_devices_count != 0) and not (
-                    data_devices_count >= min_pool_width):
+            if (data_devices_count !=
```
Comment (mssawant) [inline provisioning/miniprov/hare_mp/cdf.py:81]: I get flake errors in this case as `line break after an operator` did the build work for you?

Comment (SwapnilGaonkar7) [reply provisioning/miniprov/hare_mp/cdf.py:81]: Yes, build is passing.

Comment (SwapnilGaonkar7) [reply provisioning/miniprov/hare_mp/cdf.py:81]: In fact even if we move that operator to next light exact opposite error will be displayed(Line break occurred before a binary operator)

**Review by mssawant — APPROVED** [review]

File: provisioning/miniprov/hare_mp/cdf.py (hunk 1 above)
Comment (mssawant) [reply provisioning/miniprov/hare_mp/cdf.py:81]: Yeah, make rpm works but if you do make in `provisioning/miniprov` you will see the errors. Just make sure that `make` in `provisioning/miniprov`.

**Comment by mssawant** [comment]
@SwapnilGaonkar7 , please fix the DCO check.

**Comment by SwapnilGaonkar7** [comment]
@mssawant Please check now

**Pull request merged**